resources/home/dnanexus/run_workflows/utils/manage_dict.py

```
@@ -364,6 +366,37 @@ def add_other_inputs(
+        analysis_job_id = job_outputs_dict.get(
+            out_dir.replace("INPUT-", "").replace("-out_dir", "")
+        )
+
+        if not analysis_job_id:
+            raise KeyError(
+                (
+                    "Error trying to parse output directory to input dict."
+                    f"\nNo output directory found for given input: {out_dir}\n"
+                    "Please check config to ensure job input is in the "
+                    "format: INPUT-analysis_[0-9]-out_dir"
+                )
```

Ruff (0.8.2)

Avoid extraneous parentheses

Remove extraneous parentheses

(UP034)